### PR TITLE
Implemented time-related translations and updated Spanish ones

### PR DESCRIPTION
**Pull request by orionn333**
I added some new translations and translated time-related strings, the accent color dialog title and the title in phones list.

I also had to use a static application context, just needed to get strings from resources.

### CODE REVIEW

File: chooloolib/src/main/java/com/chooloo/www/chooloolib/BaseApp.kt [hunk 1]
```diff
@@ -8,9 +8,13 @@ import javax.inject.Inject
 
 abstract class BaseApp : Application() {
     @Inject lateinit var preferences: PreferencesInteractor
+    companion object {
+        lateinit var instance: BaseApp private set
```
Comment (roeiedri) [inline chooloolib/src/main/java/com/chooloo/www/chooloolib/BaseApp.kt:12]: Please don't use a static instance of the App

File: chooloolib/src/main/java/com/chooloo/www/chooloolib/model/ListData.kt [hunk 2]
```diff
@@ -62,7 +70,7 @@ data class ListData<DataType>(
 
         fun fromPhones(phones: List<PhoneAccount>): ListData<PhoneAccount> {
```
Comment (roeiedri) [inline chooloolib/src/main/java/com/chooloo/www/chooloolib/model/ListData.kt:71]: You can pass `context` to `fromPhones()` and use that, using a static instance of the app and in a dedicated object inside this data class is not a recommended behaviour :)

**Review by roeiedri — CHANGES_REQUESTED** [review]

File: chooloolib/src/main/java/com/chooloo/www/chooloolib/adapter/PhonesAdapter.kt [hunk 3]
```diff
@@ -26,5 +29,5 @@ class PhonesAdapter @Inject constructor(
         }
     }
 
-    override fun convertDataToListData(data: List<PhoneAccount>) = ListData.fromPhones(data)
+    override fun convertDataToListData(data: List<PhoneAccount>) = ListData.fromPhones(data, context)
```
Comment (orionn333) [inline chooloolib/src/main/java/com/chooloo/www/chooloolib/adapter/PhonesAdapter.kt:32]: First of all, I'm sorry for my mistakes (especially about creating a static instance of the app as a way to get `context`). Would the changes I have made be alright?

Comment (roeiedri) [reply chooloolib/src/main/java/com/chooloo/www/chooloolib/adapter/PhonesAdapter.kt:32]: Yes looks great, just please remove the static app cotext and the Strings object

**Comment by orionn333** [comment]
I will create a new pull request once I've cleaned up my fork

**Pull request closed without merging**